mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -911,22 +912,27 @@ Type ContractionOp::getExpectedMaskType() {
 
   unsigned numVecDims = lhsIdxMap.getNumDims();
   SmallVector<int64_t> maskShape(numVecDims, ShapedType::kDynamic);
+  SmallVector<bool> maskShapeScalabledims(numVecDims, false);
```

Comment:
`s/maskShapeScalabledims/maskShapeScalableDims`

Reply:
Ta, see the latest fixup commit